### ISSUE TITLE
Incorrect output on latest Windows 11 dev build

### ISSUE THREAD

**Issue by SimPilotAdamT** [dylanaraps/neofetch#2046]
## Description
Running neofetch on the latest dev insider build of Windows 11 returns the OS as `Windows (Unknown) x86_64` and defaults to the Windows 7 ASCII Art. 

#### Does this issue still occur in the master branch? (**Required if issue**)
Yes

#### Neofetch version
`7.1.0`

## Screenshot
![image](https://user-images.githubusercontent.com/71192298/152695838-750f2cf7-49ec-4d56-a149-c39ba9c0b08c.png)

## Config file
Default, no changes made.

## Verbose log
Uploaded to [ControlC](https://controlc.com/7ea008d2)

**Comment by Deliphin** [comment]
It seems Windows 11's logo isn't in neofetch yet. I tried `neofetch --ascii_distro Windows11` and got the same thing. (I'm on neofetch 7.1.0-2 on Arch)

**Comment by SimPilotAdamT** [comment]
> It seems Windows 11's logo isn't in neofetch yet. I tried `neofetch --ascii_distro Windows11` and got the same thing. (I'm on neofetch 7.1.0-2 on Arch)

yep no logo and no recognition of the few ppl who installed neofetch onto Win11 Using Scoop

**Comment by SimPilotAdamT** [comment]
On closer inspection, it seems to be because neofetch tries to use WMIC, when that has been long deprecated and deleted from Windows.
